subprojects/docs/src/docs/release/notes.md

```diff
@@ -102,6 +104,8 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 }
 ```
 
+This was done to ensure that version catalogs are accessible via API in situations where generated accessors are not.
```

Comment:
@eskatos, I updated the statement but it still doesn't feel very impactful.

Reply:
I'm not a fan of this, I will delete and merge.

Reply:
The new statement is more accurate. Feel free to omit though, I think it just paraphrases what's above.